PyDMXControl/controllers/_serialController.py

```diff
@@ -0,0 +1,37 @@
+"""
+ *  PyDMXControl: A Python 3 module to control DMX using uDMX.
+ *                Featuring fixture profiles, built-in effects and a web control panel.
+ *  <https://github.com/MattIPv4/PyDMXControl/>
+ *  Copyright (C) 2018 Matt Cowley (MattIPv4) (me@mattcowley.co.uk)
+"""
+
+import serial as sr
+import time as tm
```

Comment:
Unless I'm going made, time isn't being used here?

Reply:
Now it is. I missed this before. Thanks for the reminder.